packages/oui-datagrid/src/cell/cell.controller.js

```diff
@@ -0,0 +1,39 @@
+export default class {
+    constructor ($scope, $element) {
+        "ngInject";
+
+        this.$scope = $scope;
```

Comment:
Not used?

Reply:
Good catch, no more used since I pass through the datagrid to create the cell scope.